### PR TITLE
Implement Twitter timeline

### PR DESCRIPTION
**Pull request by ivanzugnoni**

### CODE REVIEW

File: twitter_timeline/twitter/views.py [hunk 1]
```diff
@@ -37,17 +40,43 @@ def home(request, username=None):
...
+
+
+@login_required()
+@require_POST
+def unfollow(request):
+    unfollowed = get_object_or_404(User, username=request.POST['username'])
+    try:
+        Relationship.objects.get(
+            follower=request.user, following=unfollowed).delete()
+    except Relationship.DoesNotExist:
```
Comment (santiagobasulto) [inline twitter_timeline/twitter/views.py]: Acá también podés usar `get_object_or_404`

File: twitter_timeline/twitter/views.py [hunk 2]
```diff
@@ -37,17 +40,43 @@ def home(request, username=None):
     else:
         form = TweetForm()
         if username is not None:
-            user = get_object_or_404(get_user_model(), username=username)
+            user = get_object_or_404(User, username=username)
```
Comment (santiagobasulto) [inline twitter_timeline/twitter/views.py]: Siempre conviene usar `get_user_model()`, incluso cuando vos lo implementás.

File: twitter_timeline/twitter/models.py [hunk 3]
```diff
@@ -9,3 +10,39 @@ class Meta:
...
+        
+    def unfollow(self, twitter_profile):
+        try:
+            rel = Relationship.objects.get(follower=self, following=twitter_profile)
+        except Relationship.DoesNotExist:
+            pass
+        rel.delete()
+    
+    def is_following(self, twitter_profile):
+        try:
```
Comment (martinzugnoni) [inline twitter_timeline/twitter/models.py]: `return Relationship.objects.filter(follower=self, following=twitter_profile).exists()`

File: twitter_timeline/twitter/views.py [hunk 4]
```diff
@@ -39,15 +42,40 @@ def home(request, username=None):
...
+        'following_profile': following_profile
     })
 
 
 @login_required()
+@require_POST
+def follow(request):
+    followed = get_object_or_404(
+        get_user_model(), username=request.POST['username'])
+    Relationship.objects.create(follower=request.user, following=followed)
```
Comment (martinzugnoni) [inline twitter_timeline/twitter/views.py]: Don't create Relationship manually, we already have that logic abstracted in User.follow().

**Comment by martinzugnoni** [comment]
Looking good so far +1

**Pull request closed without merging**